feat: add functions for create sub account, webhook validation, get banks, and bulk transfer

fikremariamF opened the pull request:
…ransfer

isrugeek commented:
Great work; I suggest updating the doc, too. 

I also don't see split-payments.

fikremariamF commented:
> Great work; I suggest updating the doc, too.
> 
> I also don't see split-payments.

The split payment functionality is already included in the initiate transaction functionality, I have updated the documentation to show that. But the create subaccount function was not there so I added it. While create subaccount the user is most likely to use get banks end point so I have added that one as well.
